credentialdigger/models/snippet_model/snippet_model.py

```diff
@@ -201,3 +202,20 @@ def _remove_initial_junk(self, snippet):
             r"^((\s*|\ *)\@\@.*\@\@(\s*|\ *)|(\s*|\ *)\+(\s*|\ *)|(\s*|\ *)\-(\s*|\ *)|(\s*|\ *))",
             "",
```

Comment:
please replace " with ' for these strings (we were not consistent before this PR but I didn't notice it)